src/components/SharingButtons.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import BaseComponent from 'lib/BaseComponent';
+
+export default class SharingButtons extends BaseComponent {
+
+  render() {
+    let renderButtons = () => {
+      let data = [];
```

Comment:
Sure I'll change that. I've used `data` as a variable for render methods in other classes. Should I change those to `let nodes = []` as well?